mantidproject/mantid
# Remove invalid context menu items for muon multi fit

We had some invalid graph context menu items appearing for muon multifit data that did not work.

This removes them.

**To test:**
1. Start Muon Analysis interface
1. Load any data (current run on any instrument should be fine)
1. Go to the settings menu
1. Ensure the Multi Fit checkbox is NOT selected
1. Go to the Data Analysis tab
1. Right click on the graph
1. The Add xxxxx ... options should be present and work
1. Back on the settings tab, change the multi fit check to true
1. Back in the data analysis tab, right click on the graph as before.
1. The Add xxxx... optins and the plot guess options should be gone
1. Close the muon interface
1. Open a graph of any data (you can use the loaded muon workspaces)
1. Open the Fit property Browser (using the toolbar button for example)
1. check that all the "missing" options are present for normal fitting

Fixes #22022

**Release Notes** 

*Does not need to be in the release notes.*


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
